server/routes/dashboard.js

```diff
@@ -25,14 +25,18 @@ router.post(['/search/reference', '/api/search/reference'],
         const logger = getLogger(req.requestId);
         try {
             const caseRef = req.form.data['case-reference'].toUpperCase();
+
+            logger.info('SEARCH_REFERENCE', { reference: caseRef });
+
             const reference = encodeURIComponent(caseRef);
 
+            logger.info('SEARCH_REFERENCE_ENCODED', { encoded_reference: reference });
```

Comment:
Could this be consolidated into a single line of logging? (I don't know much about our logging ecosystem)
```suggestion
            logger.info('SEARCH_REFERENCE_ENCODED', { encoded_reference: reference, reference: caseRef });
```

Reply:
I only added it twice incase for some reason the encoding goes bang. Shouldn't but always a chance?

Reply:
But [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) is a relatively uncontroversial and predictable bit of code, so is this extra bit of logging useful?

Reply:
Yeah I agree 👍🏼

Reply:
Changed to remove this unnecessary logging.